tests/api_tests/users/test_views.py

```diff
@@ -204,3 +205,199 @@ def test_get_projects_logged_in_as_different_user(self):
+            'orcid': self.newOrcid,
+            'researcherId': self.newResearcherId,
+        }, auth=self.auth_one)
+        assert_equal(res.status_code, 200)
+        assert_equal(res.json['data']['fullname'], self.new_fullname)
+        assert_equal(res.json['data']['given_name'], self.new_given_name)
+        assert_equal(res.json['data']['family_name'], self.new_family_name)
+        assert_equal(res.json['data']['suffix'], self.new_suffix)
+        assert_equal(res.json['data']['github'], self.newGithub)
+        assert_equal(res.json['data']['personal_website'], self.newPersonal_website)
+        assert_equal(res.json['data']['twitter'], self.newTwitter)
+        assert_equal(res.json['data']['linkedIn'], self.newLinkedIn)
+        assert_equal(res.json['data']['impactStory'], self.newImpactStory)
+        assert_equal(res.json['data']['orcid'], self.newOrcid)
+        assert_equal(res.json['data']['researcherId'], self.newResearcherId)
+
+    def test_put_user_logged_out(self):
+        res = self.app.put_json(self.user_one_url, {
+            'id': self.user_one._id,
+            'fullname': self.new_fullname,
+            'given_name': self.new_given_name,
+            'family_name': self.new_family_name,
+            'suffix': self.new_suffix,
+            'github': self.newGithub,
+            'personal_website': self.newPersonal_website,
+            'twitter': self.newTwitter,
+            'linkedIn': self.newLinkedIn,
+            'impactStory': self.newImpactStory,
+            'orcid': self.newOrcid,
+            'researcherId': self.newResearcherId,
+        }, expect_errors=True)
+        # This is 403 instead of 401 because basic authentication is only for unit tests and, in order to keep from
+        # presenting a basic authentication dialog box in the front end. We may change this as we understand CAS
+        # a little better
+        assert_equal(res.status_code, 403)
+
+    def test_put_user_not_logged_in(self):
+        # User tries to update someone else's user information via put
+        res = self.app.put_json(self.user_one_url, {
+            'id': self.user_one._id,
+            'fullname': self.new_fullname,
+            'given_name': self.new_given_name,
+            'family_name': self.new_family_name,
+            'suffix': self.new_suffix,
+            'github': self.newGithub,
+            'personal_website': self.newPersonal_website,
+            'twitter': self.newTwitter,
+            'linkedIn': self.newLinkedIn,
+            'impactStory': self.newImpactStory,
+            'orcid': self.newOrcid,
+            'researcherId': self.newResearcherId,
+        }, auth=self.auth_two, expect_errors=True)
+        # This is 403 instead of 401 because basic authentication is only for unit tests and, in order to keep from
+        # presenting a basic authentication dialog box in the front end. We may change this as we understand CAS
+        # a little better
+        assert_equal(res.status_code, 403)
```

Comment:
Might as well make an assertion that the data wasn't changed as well, just to be sure. Same with similar functions.